webrtc.html

```diff
@@ -4050,15 +4123,15 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
         <p>A new media track may be associated with an existing
         <code><a>MediaStream</a></code>. For example, if a remote peer adds a
         new <code><a>MediaStreamTrack</a></code> object to a
-        <code><a>MediaStream</a></code> that is being sent over an
-        <code><a>RTCPeerConnection</a></code>, this is observed on the local
+        <code><a>RTCPeerConnection</a></code>, and indicates that the
+        <code><a>MediaStreamTrack</a></code> is a member of a <code><a>
+        MediaStream</a></code> that has already been created locally
+        by the <code><a>RTCPeerConnection</a></code>, this is observed on the local
         user agent. If this happens for the reason exemplified, or for any
         other reason than the <code><a href=
         "getusermedia.html#dom-mediastream-addtrack">addTrack()</a></code>
-        method being invoked locally on a <code><a>MediaStream</a></code> or
-        tracks being added as the stream is created (i.e. the stream is
-        initialized with tracks), the user agent MUST run the following
-        steps:</p>
```

Comment:
I don't think the part about "tracks being added as a stream is created" should be removed.

new MediaStream(track1, track2); shouldn't fire events.